src/Centreon/Infrastructure/HostConfiguration/Repository/HostConfigurationRepositoryRDB.php

```diff
@@ -588,7 +590,7 @@ public function findHostNamesAlreadyUsed(array $namesToCheck): array
             $names[] = (string) $name;
         }
 
-        if (empty($names)) {
+        if (count($names) == 0) {
```

Comment:
what is the phpstan error here ?

Reply:
Variable $names in empty() always exists and is not falsy.

Reply:
If i use strict comparison, I get this error : Strict comparison using === between int<1, max> and 0 will always evaluate to false.

Reply:
then you can remove `if` statement, cause it cannot be empty

Reply:
done